packages/stable/src/__tests__/api/events.int.spec.ts

```diff
@@ -129,15 +129,16 @@ describe('Events integration test', () => {
     test('descending', async () => {
       await client.events.list({ sort: { endTime: SortOrder.DESC } });
     });
-    test('multiple props not supported', async () => {
+    test('multiple props supported', async () => {
```

Comment:
The events API added support for multiple fields. So the test is not valid anymore.